plugin/PDFView.qml

```diff
@@ -132,6 +136,35 @@ SilicaFlickable {
         linkColor: Theme.highlightColor;
         pagePlaceholderColor: Theme.highlightColor;
 
+        onPageLayoutChanged: if (rememberPositionConfig.value) restorePageSettings()
```

Comment:
Or would PDFDocumentPage be more natural place save/restore functionality? Rationale: PDFView would stay more as a "stupid" viewer component for content, not needing to care if multiple instances are created of same file or anything like that.